service/error_handlers.py

```diff
@@ -24,64 +24,3 @@ def bad_request(error):
       ),
       status.HTTP_400_BAD_REQUEST,
   )
-
-
-@app.errorhandler(status.HTTP_404_NOT_FOUND)
-def not_found(error):
-  """ Handles resources not found with 404_NOT_FOUND """
-  msg = str(error)
-  app.logger.warning(msg)
-  return (
-    jsonify(
-      data = [],
-      message =  msg,
-    ),
-    status.HTTP_404_NOT_FOUND,
-  )
-
-
-@app.errorhandler(status.HTTP_405_METHOD_NOT_ALLOWED)
-def method_not_supported(error):
-  """ Handles unsuppoted HTTP methods with 405_METHOD_NOT_SUPPORTED """
-  msg = str(error)
-  app.logger.warning(msg)
-  return (
-    jsonify(
-      data = [],
-      message=msg,
-    ),
-    status.HTTP_405_METHOD_NOT_ALLOWED,
-  )
-
-
-@app.errorhandler(status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
-def mediatype_not_supported(error):
-  """ Handles unsuppoted media requests with 415_UNSUPPORTED_MEDIA_TYPE """
-  msg = str(error)
-  app.logger.warning(msg)
-  return (
-    jsonify(
-      data = [],
-      message=msg,
-    ),
-    status.HTTP_415_UNSUPPORTED_MEDIA_TYPE,
-  )
-
-
-@app.errorhandler(status.HTTP_500_INTERNAL_SERVER_ERROR)
-def internal_server_error(error):
-  """ Handles unexpected server error with 500_SERVER_ERROR """
-  msg = str(error)
-  app.logger.error(msg)
-  return (
-    jsonify(
-      data=[],
-      message=msg,
-    ),
-    status.HTTP_500_INTERNAL_SERVER_ERROR,
-  )
-
-@app.errorhandler(TypeError)
```

Comment:
We might still need this one, my bad. I'm going to revert this function, and then merge. Sorry for not pointing it out in the description.